lib/hummingbird.ex

```diff
@@ -122,4 +126,18 @@ defmodule Hummingbird do
       conn.assigns[:trace_id]
     end
   end
+
+  @doc """
+  Produces a random span ID.
+
+  Produces a string of lowercase hex-encoded characters of length 16 by
+  default.
+  """
+  def random_span_id(length \\ 16) do
+    length
+    |> :crypto.strong_rand_bytes()
+    |> Base.encode16()
+    |> binary_part(0, length)
```

Comment:
this part is very similar to the code that makes phoenix secret keybases except that in that function, they use `Base.encode64/1`